openapi-docs/docs/DefaultApi.md

```diff
@@ -0,0 +1,224 @@
+# AmbianicFunctionsCollection.DefaultApi
+
+All URIs are relative to *https://33743be6-3197-4dd4-8471-50b3640320c9.mock.pstmn.io*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**notification**](DefaultApi.md#notification) | **POST** /mock-notification | notification
+[**subscribe**](DefaultApi.md#subscribe) | **POST** /mock-subscribe | subscribe
+[**subscription**](DefaultApi.md#subscription) | **DELETE** /mock-subscription | subscription
+[**subscriptionData**](DefaultApi.md#subscriptionData) | **GET** /mock-subscription | subscription-data
+
+
+
+## notification
+
+> notification(userId, notification, accessControlAllowOrigin)
+
+notification
+
+An endpoint to send notifications about detected objects to a premium ambianic subscriber from a running edge device, delivered from the Cloud API.
+
+### Example
+
+```javascript
+import AmbianicFunctionsCollection from 'ambianic_functions_collection';
+
+let apiInstance = new AmbianicFunctionsCollection.DefaultApi();
+let userId = AUTHO_USER_ID ; // String | Authenticated user ID gotten from auth0
+let notification = NOTIFICATION_DATA; // String | Notification object containing detected event details from running ambianic edge device
+let accessControlAllowOrigin = *; // String | 
+apiInstance.notification(userId, notification, accessControlAllowOrigin, (error, data, response) => {
```

Comment:
@vickywane the JS function name convention is different than REST API method naming convention. This is why we talked about using `operationId` in the API spec file. 

JS code should look like `apiInstance.sendNotification()`.

Reply:
This has been corrected by modifying the `operationId`